CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.0
```

Comment:
is this a breaking change?

Reply:
Could be. Jest was updated, and now enforces no nested specs (previously these just got ignored).

Reply:
But even if they counted it as a patch bump ;)

Reply:
It also needs the webpack changes from #164